a10_neutron_lbaas/v2/neutron_ops.py

```diff
@@ -45,3 +49,6 @@ def loadbalancer_total(self, context, tenant_id):
 
     def pool_get(self, context, pool_id):
         return self.plugin.db.get_pool(context, pool_id)
+
+    def bcm_factory(self):
+        return bcm
```

Comment:
`bcm_factory` is called as such:
https://github.com/a10networks/a10-neutron-lbaas/pull/269/files#diff-d79997eae33bb0fff59596d6b9a52cbbR21
So yes, yes it should.